documentation/src/main/asciidoc/topics/ref_rest_protobuf.adoc

```diff
@@ -0,0 +1,82 @@
+[id='rest_v2_create_protobuf']
+= Creating Protobuf Schemas
+Create Protobuf schemas across {brandname} clusters with `POST` requests that
+include the content of a protobuf file in the payload.
+
+[source,options="nowrap",subs=attributes+]
+----
+POST /rest/v2/protobuf/{schemaName}
+----
+
+If the schema already exists, {brandname} returns `CONFLICT`. If the schema is
+not valid, {brandname} stores the schema and returns the error in the response body.
```

Comment:
In order to clarify a bit further on how schemas work, I suggest:

```If the schema already exists, {brandname} returns `CONFLICT`. If the schema is
not valid, either because of syntax errors, or because some of its dependencies are missing, {brandname} stores the schema and returns the error in the response body.```

Reply:
ok